orca-bakery/src/main/groovy/com/netflix/spinnaker/orca/bakery/pipeline/BakeStage.groovy

```diff
@@ -129,7 +136,28 @@ class BakeStage implements StageDefinitionBuilder {
           return deploymentDetails
         }
       ]
+
+      // TODO: add condition for user-specified opt-out setting in addition to feature toggle
```

Comment:
you don't have to wait for the UI to be implemented for this - you can implement it right away (users can always tweak the json before the UI is ready)

Reply:
I know. This was just a note-to-self... 😄  It's on the TO-DO list above.